src/preamble.js

```diff
@@ -1069,6 +1069,7 @@ function createWasm() {
         // to load ok, but we do actually recompile the binary every time).
         var cachedCodeFile = '{{{ WASM_BINARY_FILE }}}.' + v8.cachedDataVersionTag() + '.cached';
         cachedCodeFile = locateFile(cachedCodeFile);
+        if (!nodeFS) nodeFS = require('fs');
```

Comment:
What would not normally get initialized?  Maybe a comment?

Reply:
Well, this is just a normal code pattern we have in a bunch of places. Not sure it's worth commenting everywhere.

We don't want to `require('fs')` during startup all the time, so we do it on demand. Maybe we should have a helper function `getNodeFS` or such?

Reply:
(landing this, but if we have ideas to improve that pattern lets keep discussing and follow up with that)

Reply:
sgtm.. I just wasn't' familiar with this.